app/styles/_core.less

```diff
@@ -1209,3 +1209,14 @@ copy-to-clipboard .input-group.limit-width {
     width: 100%;
   }
 }
+
+// handle top-border in table rows inside of nested loops
+tbody.nested-loop {
+  > tr {
+    + tr {
+      &:nth-child(2) {
+        border-top-width: 0;
```

Comment:
I think @sg00dwin is probably a better qualified as I know he has recently fixed some stuff in _tables.less.